[DOC] Update Traces Drilldown doc for GA (#434)
grafana/traces-drilldown: docs/sources/ui-reference/_index.md

---
description: Learn about the user interface for Traces Drilldown.
canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/investigate/
keywords:
  - Traces Drilldown
  - UI reference
title: Traces Drilldown UI reference
menuTitle: UI reference
weight: 600
---

# Traces Drilldown UI reference

> Grafana Traces Drilldown provides an easy to use interface that helps focus your tracing data exploration.

Author: I removed 'easy to use interface that helps" to just helps you focus for your tracing data exploration

Some of the screen sections are context sensitive and change depending upon the metric you've chosen.

> Refer to [Analyze tracing data](analyze-tracing-data) for more information.

Reviewer: ❗ is this link correct or should it be `../investigate/analyze-tracing-data/`

> Refer to [Analyze tracing data](analyze-tracing-data) for more information.

Author: ```suggestion
Refer to [Analyze tracing data](../analyze-tracing-data) for more information.
```

![Numbered sections of the Traces Drilldown app](/media/docs/explore-traces/traces-drilldown-screen-ui.png)

1. **Data source selection**:
   At the top left, you select the data source for your traces. In this example, the data source is set to `grafanacloud-traces`.

1. **Filters**:
   The filter bar helps you refine the data displayed.
   You can select the type of trace data, either **Root spans** or **All spans**. You can also add specific label values to narrow the scope of your investigation.

1. **Select metric type**:
   Choose between **Rate** (spans), **Errors**, or **Duration** metrics. In this example, the **Span rate** metric is selected, showing the number of spans per second.
   - The **Span rate** graph (top left) shows the rate of spans over time.
   - The **Errors** graph (top right) displays the error rate over time, with red bars indicating errors.
   - The **Duration** heatmap (bottom right) visualizes the distribution of span durations and can help identify latency patterns.

1. **Investigation-focused tabs**:
   Each metric type has its own set of tabs that help you explore your tracing data. These tabs differ depending on the metric type you've selected.

>    For example, when you are using Span rate, then the Investigation type tabs show **Breakdown**, **Service structure**, **Comparison**, and **Traces**.

Author: ```suggestion
   For example, when you use Span rate, then the Investigation type tabs show **Breakdown**, **Service structure**, **Comparison**, and **Traces**.
```

>    For example, when you are using Span rate, then the Investigation type tabs show **Breakdown**, **Service structure**, **Comparison**, and **Traces**. The **Breakdown** tab organizes attributes by their correlation with the selected metric.

Reviewer: > The **Breakdown** tab organizes attributes by their correlation with the selected metric.

Is this the **Comparison** tab?

Reviewer: Maybe we can leave this sentence out, since each tab is described in detail in the Analyze tracing data section

```suggestion
   For example, when you are using Span rate, then the Investigation type tabs show **Breakdown**, **Service structure**, **Comparison**, and **Traces**.
```
